src/components/ProfileForm/userform.js

```diff
@@ -87,10 +76,10 @@ class UserForm extends Component {
     this.setState({ [input]: e.target.value });
   };
 
-  handleDogChange = (dogs) => {
-    this.setState({dogs : dogs})
-    console.log("change userform dogs state ", this.state.dogs)
-  }
+  handleDogChange = dogs => {
+    this.setState({ dogs: dogs });
```

Comment:
you probably only need to write:

`this.setState({dogs});`

Reply:
We have multiple items in that state, not sure if we could just pass the dogs object to the state